python/flink_agents/runtime/remote_execution_environment.py

```diff
@@ -133,10 +138,26 @@ class RemoteExecutionEnvironment(AgentsExecutionEnvironment):
     """Implementation of AgentsExecutionEnvironment for execution with DataStream."""
 
     __env: StreamExecutionEnvironment
+    __config: LocalConfiguration
 
     def __init__(self, env: StreamExecutionEnvironment) -> None:
         """Init method of RemoteExecutionEnvironment."""
         self.__env = env
+        self.__config = LocalConfiguration()
+        flink_conf_dir = os.environ.get("FLINK_CONF_DIR")
+        if flink_conf_dir is not None:
+            config_dir = Path(flink_conf_dir) / "config.yaml"
```

Comment:
Does this `config.yaml` mean that flink-agents requires Flink 2.0+?

Reply:
Does Flink 1.20 support the old config file `flink-config.yaml`? If yes, do we want to support it in flink-agents for consistency?